config/routes.rb

```diff
@@ -7,4 +7,11 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  root to: 'users#index'
+
+  resources :users, only: [:index, :show] do
+  end
+
+  resources :posts, only: [:index, :show] do
+  end
```

Comment:
- [ ] Kindly correct the routes for users and posts following the example from [Ruby on Rails Guides](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router) to avoid issues. The correct URL for posts#index should be http://127.0.0.1:3000/users/1/posts/
And posts#show http://127.0.0.1:3000/users/1/posts/1

![image](https://github.com/jcmunav63/my-ror-blog/assets/58642949/02159b8b-b32f-4acc-917f-bed28453f0ec)